centreon/src/Centreon/Domain/Repository/NagiosServerRepository.php

```diff
@@ -48,72 +93,113 @@ public function getPaginationList($filters = null, int $limit = null, int $offse
     {
         $collector = new StatementCollector;
 
-        $sql = 'SELECT SQL_CALC_FOUND_ROWS * '
-            . 'FROM `' . $this->getClassMetadata()->getTableName() . '`';
+        $sql = 'SELECT SQL_CALC_FOUND_ROWS * FROM `:db`.`nagios_server`';
 
         if ($filters !== null) {
             $isWhere = false;
 
-            if (!empty($filters['search'])) {
-                $sql .= ' WHERE `' . $this->getClassMetadata()->getColumn('name') . '` LIKE :search';
+            if (! empty($filters['search'])) {
+                $sql .= ' WHERE `name` LIKE :search';
                 $collector->addValue(':search', "%{$filters['search']}%");
                 $isWhere = true;
             }
 
-            if (array_key_exists('ids', $filters) && is_array($filters['ids'])) {
+            if (
+                array_key_exists('ids', $filters)
+                && is_array($filters['ids'])
+            ) {
```

Comment:
Missing check about empty array like in another location of this PR ?
```suggestion
            if (
                array_key_exists('ids', $filters)
                && is_array($filters['ids'])
                && [] !== $filters['ids']
            ) {
```

Reply:
done ✅